docs/configuration.md

```diff
@@ -29,6 +29,10 @@ Local address to run gRPC server on (default: `"[::]:50051"`, deprecated, will b
 
 [Broadcast adapter](./broadcast_adapters.md) to use. Available options out-of-the-box: `redis` (default), `nats`, `http`.
 
+**grpc_impl** (`ANYCABLE_GRPC_IMPL`, `--grpc-impl`)
```

Comment:
It's an env-only option for now, we do not expose it via `--grpc-impl`.

For now, let's keep it hidden, it's too experimental.